PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -1813,8 +1816,6 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 				saveGUIParams(); //writeGUIParams writeScintillaParams
 				saveFindHistory(); //writeFindHistory
 				_lastRecentFileList.saveLRFL(); //writeRecentFileHistorySettings, writeHistory
-				saveProjectPanelsParams(); //writeProjectPanelsSettings
-				saveFileBrowserParam();
 				//

```

Comment:
Here I moved the parameter saving functions for the _Project Panels_ and the _Folder as Workspace_ to an earlier position. Reason: **saveProjectPanelParams** function can be cancelled now, same way as the save of the edit buffer files.